azkaban-common/src/main/java/azkaban/executor/FetchActiveFlowDao.java

```diff
@@ -98,6 +120,22 @@ public FetchActiveFlowDao(final DatabaseOperator dbOperator) {
     }
   }
 
+  /**
+   * Fetch unfinished flows similar to {@link #fetchUnfinishedFlows}, excluding flow data.
```

Comment:
@jamiesjc, do you have some practical experience on the performance impact on azkaban-web after switching to get unfinished executions from DB instead of caching? I mean situation when there are at least some, say 100 concurrent executions and those are for considerable big flows (with many nodes & properties) so that the flow data is big.

I have a vague memory that it can take long to load even a couple of flows, when the data is included.. I think I saw that in azkaban-web startup when it's initializing the cache of unfinished flows from DB, it could be something like 10 seconds. For example, if the flow updater thread is now fetching the list in a loop and it takes that long, it may become unmanageable with any higher number of parallel executions. Just suspicions so far, I hope you have some knowledge based on manual testing?

Not sure how big difference it makes to exclude the flow data like here, but maybe this should be used as much as possible and only fetch the full flow data when it's required?

Reply:
@juhoautio Good point. I've tested the performance on local mysql DB with hundreds of running flows and didn't see significant degradation. I'm about to test it in our staging environment before enabling it in production. 
I agree we should fetch full flow data only when it's needed for better DB performance. Thanks for reminding me of this. I'll share any test result I get later.